<?xml version="1.0"?>
<launch>  
<arg name="revise_angle_128" value="0.027721,0.002361,-0.043092,-0.023711,0.003442,-0.046271,-0.018255,0.036872,-0.048702,-0.025516,0.002106,-0.040973,-0.019308,0.001663,
-0.045102,-0.015489,0.001148,-0.047884,-0.018850,0.016742,-0.044528,-0.033280,0.001597,-0.039345,1,1.8,1,1.8,0.15,0.15,"/>
<arg name ="revise_angle_32" value="0, 0, 0, 0, 0, 0, 0, 0, 0,0, 0, 0,0, 0, 0,0, 0, 0,0, 0, 0,0, 0, 0,45,0,0,0,0.1,0,"/>
<arg name="read_fast" default="false" />
<arg name="read_once" default="false" />
<arg name="repeat_delay" default="0.0" />
<param name="model" value="R-Fans-32" /> <!-- R-Fans-32 R-Fans-16 C-Fans-128 -->
<node pkg="rfans_driver" type="driver_node" name="rfans_driver" output="screen">
  <param name="advertise_name" value="rfans_packets" />
  <param name="control_name" value="rfans_control"/>
  <param name="device_ip" value="192.168.0.3" />
  <param name="device_port" value="2014" />
  <param name="rps" value="10"/>
  <param name="pcap"  value=""/>
  <param name="data_level" value="3" />
  <param name="use_double_echo" value="false"/>
  <param name="read_fast" value="$(arg read_fast)" />
  <param name="read_once" value="$(arg read_once)" />
  <param name="repeat_delay" value="$(arg repeat_delay)" />
</node>
<node pkg="rfans_driver" type="calculation_node" name="calculation_node">
  <param name="advertise_name" value="rfans_points" />
  <param name="subscribe_name" value="rfans_packets" />
  <param name="frame_id" value="/world"/>
  <param name="use_gps" value="false"/>
  <param name="revise_angle_128" value="$(arg revise_angle_128)"/>
  <param name="revise_angle_32" value="$(arg revise_angle_32)"/>
</node>
</launch>
